mono/metadata/sgen-mono.c

```diff
@@ -2889,9 +2889,9 @@ sgen_client_degraded_allocation (void)
 	if (mono_atomic_load_i32 (&last_major_gc_warned) < major_gc_count) {
 		gint32 num = mono_atomic_inc_i32 (&num_degraded);
 		if (num == 1 || num == 3)
-			mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_GC, "Warning: Degraded allocation.  Consider increasing nursery-size if the warning persists.");
+			mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_GC, "Warning: Degraded allocation.  Consider increasing nursery-size if the warning persists.");
 		else if (num == 10)
-			mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_GC, "Warning: Repeated degraded allocation.  Consider increasing nursery-size.");
+			mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_GC, "Warning: Repeated degraded allocation.  Consider increasing nursery-size.");
```

Comment:
Again, I think these should stay INFO.

Reply:
Actually, maybe even WARNING? idk
Again, need Vlad's opinion. This one definitely shouldn't be DEBUG though.

Reply:
Okay, based on Vlad's comment let's make this WARNING. I'll take a second look over this, including the GC portions, later today.